### PR TITLE
Feat/login page pet clinic

### PR DESCRIPTION
**Pull request by Metemi**
HI, Just want to show what I have done. 
It is Login page.... just in seperated app.  I can than create new branch from main and create new page in pages

### CODE REVIEW

File: AvengersPetClinic/src/App.tsx [hunk 1]
```diff
@@ -1,33 +1,48 @@
...
+                type='text'
+                name='username'
+                value={username}
+                onChange={function (event){
+                  setUsername(event.target.value)
+                }}
+            ></input></label>
+          </div> 
+          <div className='inputGroup'>
+            <label className='label'> Passwor:
```
Comment (Normelik) [inline AvengersPetClinic/src/App.tsx:32]: letter D is missing

Comment (Metemi) [reply AvengersPetClinic/src/App.tsx]: Letter D added

File: AvengersPetClinic/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+          <div className='inputGroup'>
+            <label className='label'> Password: 
+                <input className='input'
+                type='password'
+                name='password' 
+                value={password}
+                onChange={function (event){
+                  setPassword(event.target.value)
+                }}
+              ></input>
```
Comment (markkovari) [inline AvengersPetClinic/src/App.tsx:41]: Noice

image

**Review by markkovari — COMMENTED** [review]
Nice

File: AvengersPetClinic/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                name='username'
+                value={username}
+                onChange={function (event){
+                  setUsername(event.target.value)
+                }}
+            ></input></label>
+          </div> 
+          <div className='inputGroup'>
+            <label className='label'> Password: 
+                <input className='input'
```
Comment (markkovari) [inline AvengersPetClinic/src/App.tsx:34]:
```suggestion
          <div className='inputGroup'>
            <label className='label'>Password: 
                <input className='input'
```

**Review by markkovari — CHANGES_REQUESTED** [review]
There should be no space there

**Pull request closed without merging**